test/unit/spec/v2/ion/i18nTransformer_spec.js

```diff
@@ -1825,6 +1825,41 @@ describe('v2/ion/i18nTransformer', function() {
     ],
   ])('should render correct error message by get key from API: %s', (key, error, expectedMessage) => {
     Bundles.login = originalLoginBundle;
+    expect(Bundles.login[key]).toBeUndefined();
     expect(getMessage(error)).toEqual(expectedMessage);
   });
+
+  it('returns an override key when present in the override map', () => {
```

Comment:
this test case was not my change https://github.com/okta/okta-signin-widget/commit/baf72db7a55bda231ade1fc1c46d14dc62df473d